cluster/addons/rbac/node-proxy-role.yaml

```diff
@@ -10,15 +10,9 @@ rules:
   - ""
   resources:
   - nodes/proxy
-  verbs:
-  - create
-  - get
-- apiGroups:
-  - ""
-  resources:
   - nodes/log
   - nodes/stats
   - nodes/metrics
   - nodes/spec
```

Comment:
@mikedanese @cjcullen do you know what the addon manager does with the old object if we rename these?